public/lib/atom-service.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import moment from 'moment';
+
+angular.module('wfAtomService', [])
+.service('wfAtomService', ['config', wfAtomService]);
+
+function wfAtomService(config) {
+  function parseMediaAtom(atom) {
+    const currentAsset = getCurrentAsset();
+    const editorUrl = getUrl();
+    const friendlyExpiryDate = moment(atom.metadata.expiryDate).format('dddd, MMMM Do YYYY');
+    const mediaAtomFields = {
+      friendlyExpiryDate: friendlyExpiryDate,
+      editorUrl: editorUrl,
+      youtubeUrl: currentAsset && `https://www.youtube.com/embed/${currentAsset.id}?showinfo=0`
```

Comment:
as part of PfP the player looks the same regardless of what query strings are given, so we can remove `showinfo`

see https://github.com/guardian/media-atom-maker/pull/409 and https://github.com/guardian/frontend/pull/16240